spec/spec_helper.rb

```diff
@@ -11,6 +11,10 @@
 # Must be explicit as activerecord is optional dependency
 require 'active_record/railtie'
 
+unless File.exist?('config/database.yml') || File.exist?(File.expand_path('~/.msf4/database.yml'))
```

Comment:
This doesn't hit everything, https://github.com/rapid7/metasploit-framework/blob/master/lib/metasploit/framework/database.rb

maybe ```unless Metasploit::Framework::Database.configurations_pathname.try(:to_path)```